src/libsync/syncjournaldb.cpp

```diff
@@ -900,21 +897,17 @@ bool SyncJournalDb::setFileRecord(const SyncJournalFileRecord &_record)
         int contentChecksumTypeId = mapChecksumType(checksumType);
         _setFileRecordQuery->reset_and_clear_bindings();
         _setFileRecordQuery->bindValue(1, QString::number(phash));
-        _setFileRecordQuery->bindValue(2, plen);
```

Comment:
Not storing the plen will break downgrading the client, would it not?

Reply:
Good point, I'll just remove that whole commit I guess.

Reply:
Removed